css/quotes.css

```diff
@@ -1,193 +1,225 @@
-body {
-    margin: 0;
-}
-
-.form {
-    margin-left: 5%;
-    margin-right: 5%;
-    max-width: 90%;
-    font-family: 'Roboto', sans-serif;
-}
-
-.form__title {
-    font-size: 22px;
-    font-weight: bold;
-    text-align: justify;
-    margin-top: 50px;
-    margin-bottom: 10px;
-    color: rgba(0, 0, 0, 0.7)
-}
-
-.form__subtitles {
-    font-weight: bold;
-    text-align: justify;
-    margin-bottom: 18px;
-    color: rgba(0, 0, 0, 0.7)
-}
-
-.hr__quote {
-    border: 0;
-    margin-left: 0%;
-    margin-right: 50%;
-    width: 90px;
-    height: 1px;
-    background-color: #0086c3;
-}
-
-.hr__details {
-    margin-bottom: 36px;
-}
-
-.form__paragraph {
-    font-size: 12px;
-    font-weight: lighter;
-    margin-left: 5%;
-    margin-right: 5%;
-    text-align: justify;
-    margin-bottom: 40px;
-}
-
-
-/*
+                                                body {
+                                                    margin: 0;
+                                                }
+
```

Comment:
please erase this empty line

Reply:
@Sonny97 , 1 empty line between css blocks is correct, thats what is expected

Reply:
Is it really right? Because I thought that would make the code look longer or something like that?